src/dashboards/videos/common.libsonnet

```
@@ -0,0 +1,90 @@
+local grafana = import 'grafonnet/grafana.libsonnet';
+local template = grafana.template;
+
+{
+  field: {
```

Maybe we should use the plural form (for all fields), WDYT?

```suggestion
  fields: {
```

---

Good Idea) Thanks) Updated using the plural form)